recipe/meta.yaml

```diff
@@ -40,9 +42,13 @@ test:
 
 about:
   home: https://github.com/agoose77/hatch-nodejs-version
-  summary: Hatch plugin for versioning from a package.json file
+  dev_url: https://github.com/agoose77/hatch-nodejs-version/tree/master
```

Comment:
Can you make the dev_url more generic?
https://github.com/agoose77/hatch-nodejs-version/

Reply:
Done!